go/vt/sqlparser/sql.y

```diff
@@ -942,6 +1001,152 @@ value_expression:
   {
     $$ = &ValuesFuncExpr{Name: $3}
   }
+| CONVERT openb expression ',' convert_type closeb
```

Comment:
Can you also support `CONVERT(foo USING bar)`? In terms of the second argument in that syntax, you can probably just use the existing charset grammar (i.q. sql_id) https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_convert. We don't really care about the actual contents of the arguments here except to properly re-represent them in the Format function of the ast.